src/Uno.UI.Wasm/tsBindings/WindowManageregisterUIElementReturn.ts

```diff
@@ -0,0 +1,10 @@
+/* TSBindingsGenerator Generated code -- this code is regenerated on each build */
+class WindowManageRegisterUIElementReturn
+{
+	/* Pack=4 */
+	RegistrationId : number;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class property 'RegistrationId' must be marked either 'private', 'public', or 'protected'](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5247895)